src/app/admin/sidemenu/sidemenu.component.html

```diff
@@ -12,16 +12,16 @@
             <li class="menu" routerLinkActive="active-link"  *ngIf="isSchemaCreated">
                 <a class="a-menu fs-14 p-3  d-block" routerLink="/schema"><b>{{'SCHEMA' | translate}}</b></a>
             </li>
-            <li class="menu" routerLinkActive="active-link">
-                <a class="a-menu fs-14 p-3  d-block" routerLink="/a"><b>{{'ATTESTATION_WORKFLOW' | translate}}</b></a>
+            <li class="menu" routerLinkActive="active-link" >
+                <a class="a-menu fs-14 p-3  d-block" type="button" (click)="navigateTo(ACTION_ATTESTATION)"><b>{{'ATTESTATION_WORKFLOW' | translate}}</b></a>
 
             </li>
             <li class="menu" routerLinkActive="active-link">
-                <a class="a-menu fs-14 p-3  d-block" routerLink="/a"><b>{{'VERIFIABLE_CREDENTIALS' | translate}}</b></a>
+                <a class="a-menu fs-14 p-3  d-block" type="button" (click)="navigateTo(ACTION_VC)"><b>{{'VERIFIABLE_CREDENTIALS' | translate}}</b></a>
             </li>
 
             <li class="menu" routerLinkActive="active-link">
-                <a class="a-menu fs-14 p-3  d-block" routerLink="/a"><b>{{'OWNERSHIP' | translate}}</b></a>
+                <a class="a-menu fs-14 p-3  d-block" type="button" (click)="navigateTo(ACTION_OWNERSHIP)"><b>{{'OWNERSHIP' | translate}}</b></a>
```

Comment:
@suvarnakale 
Instead of creating  below variable
 ACTION_ATTESTATION = 'attestation';
  ACTION_VC = 'vc';
  ACTION_OWNERSHIP = 'ownership';

you can  directly pass tab form here  (click)="navigateTo('attestation')"

and store value it in  currentTab  variable

Reply:
yeah.. thats doable... will update PR